### DISCUSSION TITLE
Some projects to be included

### DISCUSSION THREAD

**Discussion by achetronic** [avelino/awesome-go#5018] [Ideas]
Hello there. This is a great repository :)

I would like to suggest some repos I maintain that I think could be cool to see here:

* github.com/prosimcorp/reforma
* github.com/prosimcorp/replika
* github.com/prosimcorp/rabbit-stalker
* github.com/achetronic/nsmurder
* github.com/achetronic/wizgo
* github.com/achetronic/autoheater

Some of them are projects that manage kubernetes' related stuff, sume others are dedicated to IoT

**Comment by avelino** [comment]
to propose new projects you need to open pull requests including them, remembering that there must be a pull request for each project

[I recommend reading](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md)

**Reply by achetronic** [reply]
Hey, thank you for the answer :)
Have opened some PRs
